lib/config/presets/gitea/index.ts

```diff
@@ -16,11 +16,14 @@ export const Endpoint = 'https://gitea.com/api/v1/';
 export async function fetchJSONFile(
   repo: string,
   fileName: string,
-  endpoint: string
+  endpoint: string,
+  packageTag: string
```

Comment:
```suggestion
  packageTag?: string
```
I think this should be optional.